client/state/selectors/test/should-show-launchpad-first.ts

```diff
@@ -14,19 +14,48 @@ beforeEach( () => {
 } );
 
 describe( 'shouldShowLaunchpadFirst', () => {
-	it( 'should return true when site was created via onboarding flow and assigned to experiment', async () => {
+	it( 'should return true when site was created via onboarding flow, has an intent, and assigned to experiment', async () => {
 		( loadExperimentAssignment as jest.Mock ).mockResolvedValue( {
 			variationName: 'treatment_cumulative',
 		} );
 		const site = {
 			options: {
 				site_creation_flow: 'onboarding',
+				site_intent: 'sell',
 			},
 		} as SiteDetails;
 
 		expect( await shouldShowLaunchpadFirst( site ) ).toBe( true );
 	} );
 
+	it( 'should return false when site was created via onboarding flow, has the ai-assembler intent, and assigned to experiment', async () => {
+		( loadExperimentAssignment as jest.Mock ).mockResolvedValue( {
+			variationName: 'treatment_cumulative',
+		} );
```

Comment:
@Automattic/quake, is the experiment assignment mocking still needed in these tests? If not, I think it would be well worth cleaning up the tests.

Reply:
I left it in because I imagined there would be future experiments that would impact the launchpad. But with the change in focus and the goal-based launchpad projects on hold I can't see it being needed.

It should be cleaned up.